package.json

```diff
@@ -42,36 +41,46 @@
   },
   "homepage": "https://github.com/ForgeRock/forgerock-javascript-sdk#readme",
   "devDependencies": {
-    "@nrwl/cli": "^13.3.9",
-    "@nrwl/eslint-plugin-nx": "^13.3.9",
-    "@nrwl/express": "^13.3.9",
-    "@nrwl/jest": "^13.3.9",
-    "@nrwl/js": "^13.3.9",
-    "@nrwl/linter": "^13.3.9",
-    "@nrwl/node": "^13.3.9",
-    "@nrwl/nx-cloud": "^13.0.0",
-    "@nrwl/react": "^13.3.9",
+    "@angular-devkit/schematics": "^14.0.0-next.4",
+    "@babel/core": "^7.17.5",
+    "@nrwl/cli": "13.8.4",
+    "@nrwl/cypress": "13.8.4",
+    "@nrwl/eslint-plugin-nx": "13.8.4",
+    "@nrwl/express": "13.8.4",
+    "@nrwl/jest": "13.8.4",
+    "@nrwl/js": "13.8.4",
+    "@nrwl/linter": "13.8.4",
+    "@nrwl/node": "13.8.4",
+    "@nrwl/nx-cloud": "^13.1.6",
+    "@nrwl/react": "13.8.4",
     "@nrwl/schematics": "^8.12.11",
-    "@nrwl/tao": "^13.3.9",
-    "@nrwl/web": "^13.3.9",
-    "@nrwl/workspace": "^13.3.9",
+    "@nrwl/tao": "13.8.4",
+    "@nrwl/web": "13.8.4",
+    "@nrwl/workspace": "13.8.4",
+    "@nxext/svelte": "^13.3.0",
+    "@nxext/sveltekit": "^13.0.0-next.2",
+    "@nxext/vitest": "^13.0.0",
     "@playwright/test": "^1.17.1",
+    "@sveltejs/vite-plugin-svelte": "^1.0.0-next.37",
     "@swc/cli": "^0.1.55",
     "@swc/core": "^1.2.145",
     "@swc/helpers": "^0.3.5",
     "@testing-library/react": "12.1.2",
     "@testing-library/react-hooks": "7.0.2",
-    "@types/express": "^4.17.0",
+    "@types/express": "4.17.13",
     "@types/jest": "^27.0.0",
     "@types/jest-dev-server": "^5.0.0",
     "@types/react": "17.0.30",
     "@types/react-dom": "17.0.9",
-    "@typescript-eslint/eslint-plugin": "5.3.1",
-    "@typescript-eslint/parser": "5.3.1",
+    "@typescript-eslint/eslint-plugin": "5.10.2",
+    "@typescript-eslint/parser": "5.10.2",
     "@typescript-eslint/typescript-estree": "^4.7.0",
+    "@vitest/ui": "^0.5.9",
     "babel-jest": "27.2.3",
+    "c8": "^7.11.0",
     "concurrently": "^6.4.0",
-    "eslint": "8.2.0",
+    "cypress": "^9.1.0",
```

Comment:
Do we need Cypress for something at this moment?

Reply:
This may be left over, good catch